qiskit/circuit/quantumregister.py

```diff
@@ -15,61 +15,9 @@
 """
 Quantum register reference object.
 """
-import itertools
+from qiskit._accelerate import circuit
 
-from qiskit.circuit.exceptions import CircuitError
-
-from .register import Register
-from .bit import Bit
-
-
-class Qubit(Bit):
-    """Implement a quantum bit."""
-
-    __slots__ = ()
-
-    def __init__(self, register=None, index=None):
-        """Creates a qubit.
-
-        Args:
-            register (QuantumRegister): Optional. A quantum register containing the bit.
-            index (int): Optional. The index of the bit in its containing register.
-
-        Raises:
-            CircuitError: if the provided register is not a valid :class:`QuantumRegister`
-        """
-
-        if register is None or isinstance(register, QuantumRegister):
-            super().__init__(register, index)
-        else:
-            raise CircuitError(
-                f"Qubit needs a QuantumRegister and {type(register).__name__} was provided"
-            )
-
-
-class QuantumRegister(Register):
-    """Implement a quantum register."""
-
-    # Counter for the number of instances in this class.
-    instances_counter = itertools.count()
-    # Prefix to use for auto naming.
-    prefix = "q"
-    bit_type = Qubit
-
-
-class AncillaQubit(Qubit):
-    """A qubit used as ancillary qubit."""
-
-    __slots__ = ()
-
-    pass
-
-
-class AncillaRegister(QuantumRegister):
-    """Implement an ancilla register."""
-
-    # Counter for the number of instances in this class.
-    instances_counter = itertools.count()
-    # Prefix to use for auto naming.
-    prefix = "a"
-    bit_type = AncillaQubit
+Qubit = circuit.Qubit
```

Comment:
This is kind of a weird pattern, normally you'd only do this if you needed to rename the object. You can just import the objects. If lint complains I would add an `__all__` to say explicitly the module exports the classes.

Reply:
I got rid of the stub files in 508751975.